changedetectionio/templates/watch-overview.html

```diff
@@ -27,18 +27,18 @@
 					{% endif %}
 				{% endfor %}
 			</div>
- -->			<div id="controls-top">
+			<div id="controls-top">
 				<ul id="post-list-buttons-top">
 					<span id="checkbox-functions" style="display: none;">
```

Comment:
span dont belong in <UL>

Reply:
Removed by reworking layout with divs and adjusting css.